README.md

```diff
@@ -75,33 +16,47 @@ In order to build the project, we need to run the following command:
 
 the build generated will be placed in `dist` folder.
 
-## Tests and checks
+## Tests
 
 ### Pre-requisites
+
 Make sure that you install all the project dependencies
 
 `yarn install` or `npm install`
 
 Keep consistent, whether you use yarn or npm
 
-### Running tests in CICD pipeline
+### Run all tests
+
+`yarn test` or `npm run test`
+
+### Run unit tests
+
+`yarn test:unit` or `npm run test:unit`
+
+### Run integration tests
 
-`yarn test:ci` or `npm run test:ci`
+`yarn test:integration` or `npm run test:integration`
 
-### Test coverage
+### Run test coverage
 
 `yarn test:coverage` or `npm run test:coverage`
 
-### Format checks
-Making sure that the code format is following the guidelines
+## Format and lint
 
-`yarn format:check` or `npm run format:check`
+### Run formatter
 
-### Lint checks
-Running a linting check on the code
+`yarn format` or `npm run format`
 
-`yarn eslint:check` or `npm run eslint:check`
+### Run linter
 
+Running a linting check on the code:
+
+`yarn lint` or `npm run lint`
```

Comment:
It is a well-detected warning, following what the typescript-eslint plugin mentions in [https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-explicit-any.md](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-explicit-any.md).

In my opinion, the use of "any" is detecting, which is the only one in the code, is justified since we have to provide a mechanism for the Use Case base interface to accept as many parameters of any type as the implementations need. See the line: [https://github.com/IQSS/dataverse-client-javascript/blob/ba18a0a58a5648328382c35bbd2d863ade94d415/src/core/domain/useCases/UseCase.ts#L2](https://github.com/IQSS/dataverse-client-javascript/blob/ba18a0a58a5648328382c35bbd2d863ade94d415/src/core/domain/useCases/UseCase.ts#L2).

I couldn't find any other way to implement this, other than using "any". I'll be happy to refactor it if we find a better approach.

Thanks for noticing.